src/components/propositionalLogic/example.json

```diff
@@ -0,0 +1,50 @@
+{
+    "example1": [
```

Comment:
I just saw here that you already got to #99. In the other PR I just named the file examples.json with the plural form I think would be more fitting since they are multiple examples. If you remove the boolean values and bring them in the form of the examples.json from PR #84 then Issue #99 will be solved.